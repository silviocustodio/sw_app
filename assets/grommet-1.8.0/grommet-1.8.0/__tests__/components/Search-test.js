// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP

import React from 'react';
import renderer from 'react-test-renderer';

import Search from '../../src/js/components/Search';

// needed because this:
// https://github.com/facebook/jest/issues/1353
jest.mock('react-dom');

describe('Search', () => {
  it('has correct default options', () => {
    const component = renderer.create(
      <Search />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
