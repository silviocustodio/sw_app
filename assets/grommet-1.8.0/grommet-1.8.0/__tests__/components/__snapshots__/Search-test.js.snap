// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search has correct default options 1`] = `
<button
  aria-label={undefined}
  className="grommetux-button grommetux-button--plain"
  disabled={false}
  href={undefined}
  id={undefined}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onMouseDown={[Function]}
  onMouseUp={[Function]}
  type="button"
>
  <span
    className="grommetux-button__icon"
  >
    <svg
      aria-label="search"
      className="grommetux-control-icon grommetux-control-icon-search grommetux-control-icon--responsive"
      height="24px"
      role="img"
      version="1.1"
      viewBox="0 0 24 24"
      width="24px"
    >
      <path
        d="M15,15 L22,22 L15,15 Z M9.5,17 C13.6421356,17 17,13.6421356 17,9.5 C17,5.35786438 13.6421356,2 9.5,2 C5.35786438,2 2,5.35786438 2,9.5 C2,13.6421356 5.35786438,17 9.5,17 Z"
        fill="none"
        stroke="#000"
        strokeWidth="2"
      />
    </svg>
  </span>
</button>
`;
